### ISSUE TITLE
unknown_fields json output issue

### ISSUE THREAD

**Issue by JimjunZhang** [stephenberry/glaze#1360]
Hello,

I find a minor issue on unknow_field json output.
Here is the code:

struct unknown_fields_member
{
    std::shared_ptr\<std::string\> a = nullptr;
    std::shared_ptr\<std::string\> missing = nullptr;
    std::shared_ptr\<std::string\> end = nullptr;
    std::map\<glz::sv, glz::raw_json\> extra;
};

template \<\>
struct glz::meta\<unknown_fields_member\>
{
    using T = unknown_fields_member;
    static constexpr auto value = object("a", &T::a, "missing", &T::missing, "end", &T::end);
    static constexpr auto unknown_write{ &T::extra };
    static constexpr auto unknown_read{ &T::extra };
};

// Use
std::string buffer = R"({"unk":"zzz", "unk2":{"sub":3,"sub2":[{"a":"b"}]},"unk3":[])";
glz::context ctx{};
glz::read < glz::opts{ .error_on_unknown_keys = false } > (obj, buffer, ctx);
// encodes output string
std::string out{};
glz::write_json(obj, out);

I can see the read works well, but the write_json output is wrong, i.e. additional "," is outputted.
The out string is:   {,"unk":"zzz","unk2":{"sub":3,"sub2":[{"a":"b"}]},"unk3":[]}

Is it a bug?  any workaround to make the output works?  Thanks!

**Comment by stephenberry** [comment]
Thank for reporting this! I'll try to address it soon.

**Comment by stephenberry** [comment]
Okay, I've been able to debug this and uncover the problem. The issue is that `skip_null_members` is set to `true` and your struct is all null pointers (null members). If you set `skip_null_members = false`, then it will write out these null members and solve the problem. However, I've solved the problem and will merge in a solution soon, so that you don't need to use the option  `skip_null_members = false`.
